ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -123,6 +123,10 @@ releases:
     integration:
       name: "4.9"
       namespace: ocp
```

Comment:
seems redundant after that change

Reply:
As decided to have both 4.10 and 4.9 jobs, it will need to stay

Reply:
Removed `initial` as not used by any job